tools/dockerfile/test/cxx_gcc_7_x64/Dockerfile

```diff
@@ -58,7 +58,7 @@ RUN python3 -m pip install twisted
 RUN apt-get update && apt-get install -y cmake && apt-get clean
 
 #=================
-# Install ccache (use slighly older release of ccache that works older compilers and cmake)
```

Comment:
I think this change changes the MD5 hash so CI get's worried the image needs to be rebuilt... Please revert this change.